config_portal/frontend/README.md

```diff
@@ -0,0 +1,51 @@
+# templates/spa
```

Comment:
Do we need this file? Looks like boilerplate.

Reply:
Removed now